<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="displayBox">
      <layout class="QHBoxLayout" name="controlBox">
       <item>
        <widget class="QWidget" name="videoBox" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,0,1">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="addressLabel">
              <property name="text">
               <string>网络视频地址：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ipLineEdit">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>ip地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="portLineEdit">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>网络端口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="TV">
            <property name="minimumSize">
             <size>
              <width>960</width>
              <height>540</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="commandBox">
         <layout class="QVBoxLayout" name="controlBox_2" stretch="1,4,1">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="statusDisplayBox">
            <property name="title">
             <string>状态显示</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>10</number>
             </property>
             <item row="1" column="1">
              <widget class="QLCDNumber" name="HLcd"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="angleLabel">
               <property name="text">
                <string>CO2浓度</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>TVOC浓度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="accLablel">
               <property name="text">
                <string>湿度</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLCDNumber" name="TVOCLcd"/>
             </item>
             <item row="0" column="1">
              <widget class="QLCDNumber" name="TLcd"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="speedNowLable">
               <property name="text">
                <string>温度</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLCDNumber" name="CO2Lcd"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="TVTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
